### ISSUE TITLE
REI showing incorrect recipe for Hellshelf, replacing with Andesite

### ISSUE THREAD

**Issue by D-Molohon** [Safrodev/Apotheosis-Fabric#25]
Unsure where the confliction is, but in our pack Apotheosis is showing the incorrect items used for the hellshelf recipe - and will instead replace the Hellshelf's need for a regen potion with Andesite instead. I've included screenshots with all of the instances of this that I can find for the shelves. I've also included my [latest.log](https://github.com/Safrodev/Apotheosis-Fabric/files/9145926/latest.log) for the current mod list in our pack.

The recipe _does_ work with the correct potion, but REI will autofill with Andesite, and won't execute the craft (because it's the wrong recipe). Let me know if you want/need any more info from my end!

![image](https://user-images.githubusercontent.com/96502484/179880686-098f22ce-149d-4c7d-8cd9-dc1c8bf52943.png)
![image](https://user-images.githubusercontent.com/96502484/179880700-b6f09f5a-d703-488b-aa77-321f48e2f37a.png)
![image](https://user-images.githubusercontent.com/96502484/179880723-6fb2e430-75a6-4dd4-9903-d5cb35910c7d.png)
![image](https://user-images.githubusercontent.com/96502484/179880752-cec9a3a3-7052-4f53-a7d1-1f8433ab6400.png)
![image](https://user-images.githubusercontent.com/96502484/179880811-a2e06fd3-bfd7-4f4c-bc11-033d38e3a53a.png)
![image](https://user-images.githubusercontent.com/96502484/179880832-48037b38-8b03-447b-b3ee-d0e01556ce7b.png)

Thank you for your help!

**Comment by sjs262** [comment]
This is very similar to the issue that I posted but I think he may have fixed it in the update he pushed today

**Comment by D-Molohon** [comment]
Currently on version 2.83 - still not fixed :(

**Comment by Safrodev** [comment]
Hmm this is issue is strange because the recipes display correctly for me. I'm honestly unsure why this is happening. Are you playing on a server? It may be caused by some syncing issues.

**Comment by D-Molohon** [comment]
Yes - we're playing on a server, I just pushed 2.83 to it as soon as I saw it to see if it'd resolve the issue (tried 2.82, but we kept crashing and then you fixed that lol)

My single player world is showing the correct recipe.

Any way I could alleviate it? Is it REI, or something else conflicting you think?

**Comment by dbock1989** [comment]
So I came today to advise, I have the same issue, also using it on a server.
